src/switch_core_io.c

```diff
@@ -993,6 +993,8 @@ SWITCH_DECLARE(switch_status_t) switch_core_session_read_frame(switch_core_sessi
 
   even_more_done:
 
+	switch_log_printf(SWITCH_CHANNEL_SESSION_LOG(session), SWITCH_LOG_DEBUG, "Read codec addr: %p, codec read mutex addr: %p\n", (void *)&session->read_codec, (void *)&session->codec_read_mutex);
```

Comment:
This should be ERROR right?

Reply:
TJ has requested to have these logs be DEBUG; i can turn them into ERROR if the consensus says so